src/main/java/cloud/eppo/logging/Assignment.java

```diff
@@ -33,6 +38,12 @@ public Assignment(
     this.subjectAttributes = subjectAttributes;
     this.extraLogging = extraLogging;
     this.metaData = metaData;
+
+    variationKeyString = subject + sep + featureFlag + sep + variation;
```

Comment:
In our JavaScript SDK we also include the allocation, although I'm not sure why 😅